docs/1.0/creator-tools/mapping-facial-expressions.md

```diff
@@ -67,6 +67,26 @@ Since this method only uses the concept of mouth open/closed, it works great for
 
 * Texture sizes that are square and sized in multiples of 2 (e.g., 128, 256, 512, 1024, 2048) work more optimally in the VR engine.
 
+## Animation mapping
+
+1\. Open your character's Unity scene and select the character in the Hierarchy pane.
+
+2\. In the Inspector pane, change the **Expression Type** setting to **Animation**.
+
+3\. Scroll down a bit and click **Setup Animator**. This will create an animator on the character's head that can be used to create facial animations.
+
+4\. Click on the Expression Animator reference, this will highlight the head GameObject in the scene. Select the head to begin creating animations.
```

Comment:
Edit: "Click on the Expression Animator reference, which will highlight the head's GameObject in the scene. Select the head to begin creating animations."

Reply:
Can you also bold Expression Animator, as well as a few references below too ("click \*\*Create\*\* in the", "select the \*\*Controller\*\* in the head").

These should also get some screenshots, particularly for #3, #6, #7, and maybe #8.